### ISSUE TITLE
Deprecate landuse=churchyard, replace it by a synonym in landuse=religious and amenity=grave_yard

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#7187]
This tag is problematic as it duplicates more common ways to map such objects and conflicts with some uses of this word.

Currently it is a sole tag found after entering "churchyard" in https://preview.ideditor.com/master/

See https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dchurchyard https://lists.openstreetmap.org/pipermail/tagging/2019-December/049875.html https://github.com/gravitystorm/openstreetmap-carto/issues/3991

**Comment by quincylvania** [comment]
@matkoniecz Thanks for bringing this up, I agree the tag is redundant. For the time being I just made the preset unsearchable to discourage people from adding more of these. Once consensus is reached on the deprecation and the proper upgrade path we can also flag these as outdated.

**Comment by matkoniecz** [comment]
Thanks! I would also make churchyard a synonym for both landuse=religious and amenity=grave_yard (so searching "churchyard" will show this two)

**Comment by matkoniecz** [comment]
Seems to be fully fixed now, but there is

>  quincylvania added the waitfor-consensus label 3 days ago

What else should be potentially changed? Validator rule requesting to change to either `landuse=religious` or `amenity=grave_yard`?

**Comment by quincylvania** [comment]
> What else should be potentially changed? Validator rule requesting to change to either landuse=religious or amenity=grave_yard?

@matkoniecz Yes, I'd like to add an upgrade path once there's consensus this is considered deprecated. Mappers can get confused when they see existing features iD appear to support but they can't add themselves.
